processor/spanmetricsprocessor/processor.go

```diff
@@ -246,7 +246,8 @@ func (p *processorImp) collectCallMetrics(ilm *pdata.InstrumentationLibraryMetri
 
 		mCalls := pdata.NewMetric()
 		mCalls.SetDataType(pdata.MetricDataTypeIntSum)
-		mCalls.SetName("calls")
+		mCalls.SetName("calls_total")
+		mCalls.IntSum().SetIsMonotonic(true)
```

Comment:
Please add a test for this. I think it should be in `verifyConsumeMetricsInput`.